Updated the cryptosuite sections, mostly the Selective Disclosure part

iherman opened the pull request:
Hopefully I got it right now...


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-overview/pull/16.html" title="Last updated on Oct 17, 2024, 12:59 PM UTC (0869af9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-overview/16/73c2c35...0869af9.html" title="Last updated on Oct 17, 2024, 12:59 PM UTC (0869af9)">Diff</a>

dlongley commented on index.html:
```diff
@@ -910,20 +915,36 @@ <h5>Selective Disclosure Schemes</h5>
+              Using that information, the issuer's transformation step in the <a href="#di-integrity-structure">general data integrity structure</a> produces a
+              data structure containing the hash of the proof options (much like in the case of the full disclosure schemes) and of the set of mandatory claims, plus the identification of mandatory claims. 
+              Some elements of this data structure are then signed, the resulting structure is converted into CBOR, and 
+              encoded to provide as a Multikey formatted `proofValue`. 
```
```suggestion
              encoded to provide as a multibase-formatted `proofValue`. 
```

iherman replied:
Oops...

Wind4Greg reviewed (changes requested):
Hi Ivan, looks pretty good but a couple things are confusing to read or not quite accurate. (1) You use the term "[general data integrity structure](https://pr-preview.s3.amazonaws.com/w3c/vc-overview/pull/16.html#di-integrity-structure) when it seems that this is a "procedure" that produces a "structure". (2) The use of a "challenge" is not specific to SD schemes, and in fact BBS would use a different approach to achieving the same thing.

iherman commented:
Thank you @Wind4Greg 

> (1) You use the term "[general data integrity structure](https://pr-preview.s3.amazonaws.com/w3c/vc-overview/pull/16.html#di-integrity-structure) when it seems that this is a "procedure" that produces a "structure". 

Right. This was the result of a poor choice for the title of section 4.2.1.; I have changed that and made the changes in the references. I hope it is better now.

> (2) The use of a "challenge" is not specific to SD schemes, and in fact BBS would use a different approach to achieving the same thing.

I removed the term and reformulated/removed the sentences that used it.

I hope it is better now!

iherman commented on index.html:
```diff
@@ -312,7 +312,7 @@ <h2>High Level View of the Specifications</h2>
 
         <figure id="overall">
-          <img style="margin: auto; display: block; width: 100%;" src="diagrams/VCWG_specifications.svg" alt="Diagram illustrating the structure of the Verifiable Credentials (VCs) specification, using boxes labeled with references to the specification. Arrows on the diagram connect some of the boxes to represent (normative) dependencies. The box labeled as the VC Data Model is at the center of t ...
+          <img style="margin: auto; display: block; width: 100%;" src="diagrams/VCWG_specifications.svg" alt="Diagram illustrating the structure of the Verifiable Credentials (VCs) specification, using boxes labeled with references to the specification. Arrows on the diagram connect some of the boxes to represent (normative) dependencies. The box labeled as the VC Data Model is at the center of t ...
```
See https://github.com/w3c/vc-overview/pull/17. The PR was closed by the time you made this comment...
